docs/modules/ROOT/pages/kubernetes-platform-support.adoc

```diff
@@ -0,0 +1,20 @@
+= Supported Platform Versions
```

Comment:
I think we can move this one to the Overview page:
https://docs.hazelcast.com/operator/latest/
before `Community support` section:
https://docs.hazelcast.com/operator/latest/#community-support